lectures/inflation_history.md

```diff
@@ -4,21 +4,24 @@ jupytext:
     extension: .md
     format_name: myst
     format_version: 0.13
-    jupytext_version: 1.15.1
+    jupytext_version: 1.16.1
 kernelspec:
   display_name: Python 3 (ipykernel)
   language: python
   name: python3
 ---
 
++++ {"user_expressions": []}
```

Comment:
@Jiarui-ZH please delete this markup from jupytext?

Reply:
I have resolved this!